src/zh/jinmantiantang/src/eu/kanade/tachiyomi/extension/zh/jinmantiantang/ScrambledImageInterceptor.kt

```diff
@@ -41,7 +41,9 @@ object ScrambledImageInterceptor : Interceptor {
             return BigInteger(1, md.digest(input.toByteArray())).toString(16).padStart(32, '0')
         }
 
-        return if (aid >= 268850) {
+        return if (aid >= 421925) {
+            2 * (md5(aid.toString() + imgIndex).last().toInt() % 8) + 2
+        } else  if (aid >= 268850) {
```

Comment:
```suggestion
        } else if (aid >= 268850) {
```
Remove this extra space.